tests/app/public_contracts/schemas/GET_notifications_return.json

```diff
@@ -14,7 +14,14 @@
     },
     "links": {
       "type": "object",
-      "additionalProperties": false
```

Comment:
do we want this to be removed? what other properties would we have other than last or next?